### PR TITLE
Remove Command/Query code

### PR DESCRIPTION
**Pull request by shatl**

### CODE REVIEW

File: Solutions/SharpArch.NHibernate/SessionFactoryAttribute.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@
     ///     communicate with the database.  This allows you to declare different repositories to 
     ///     communicate with different databases.
     /// </summary>
-    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
```
Comment (seif) [inline Solutions/SharpArch.NHibernate/SessionFactoryAttribute.cs]: @shalt why this change?

File: Samples/TardisBank/Solutions/Suteki.TardisBank.Tests/ServiceLocatorInitializer.cs [hunk 2]
```diff
@@ -23,12 +19,6 @@ public static void Init()
                         .ImplementedBy(typeof(EntityDuplicateChecker))
                         .Named("entityDuplicateChecker"));
 
-            container.Register(
-                    Component
-                        .For(typeof(ISessionFactoryKeyProvider))
```
Comment (seif) [inline Samples/TardisBank/Solutions/Suteki.TardisBank.Tests/ServiceLocatorInitializer.cs]: @shatl what are those changes for nhibernate?

Comment (shatl) [inline Samples/TardisBank/Solutions/Suteki.TardisBank.Tests/ServiceLocatorInitializer.cs]: SessionKey provider is not required now

Regards, Vitaliy

File: Solutions/SharpArch.NHibernate/SessionFactoryAttribute.cs (hunk 1 above)
Comment (shatl) [inline Solutions/SharpArch.NHibernate/SessionFactoryAttribute.cs]: This is default value,  no need to duplicate it it code.

Regards, Vitaliy

**Comment by seif** [comment]
Yay, thanks Vitaliy
